### PR TITLE
Remove tslint in favor of eslint

### PR DESCRIPTION
**Pull request by mlwarner**
Updating with guidance in https://code.visualstudio.com/api/advanced-topics/tslint-eslint-migration

Options are taken from https://github.com/microsoft/vscode-extension-samples/blob/master/helloworld-sample/.eslintrc.js with the addition of the jest configs already present in this package.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -3,35 +3,20 @@ module.exports = {
     es6: true,
     node: true,
   },
+  root: true,
+  parser: "@typescript-eslint/parser",
+  plugins: ["@typescript-eslint", "jest"],
   extends: [
-    'airbnb-base',
```
Comment (mlwarner) [inline .eslintrc.js:7]: I don't feel strongly about any of these options if we want to keep the existing ones. Just reading up on the VSCode extension API and thought I could contribute some of the suggestions back.

The main goal is to enable eslint in the editor when working on this extension

**Comment by hannut91** [comment]
I also agree to use eslint instead of tslint.

**Comment by maqiv** [comment]
What is the reason that this PR has been withheld until now?

**Comment by hannut91** [comment]
It's my fault.. Sorry I'll check it soon. 😇

**Pull request closed without merging**